src/main/java/creativei/controller/InquiryController.java

```diff
@@ -57,6 +59,12 @@ ResponseObject updateInquiry(@RequestBody String inquiryStr, HttpServletRequest
             return ResponseObject.getResponse(ExceptionType.GENERAL_ERROR.getMessage(), ExceptionType.GENERAL_ERROR.getCode());
         }
     }
+    @RequestMapping(value = "/inquiry/server-info",produces = "application/json",method = RequestMethod.GET)
+    public @ResponseBody
+     ResponseObject getEnum(){
+         ResponseObject responseObject=inquiryManager.getAllEnum();
+         return responseObject;
+    }
 
     private Boolean nullStringValidation(String inquiryStr){
```

Comment:
you are also checking null and empty so it should be emptyStringValidation. 
Why do we need this method?

Reply:
to check if input string is null or not.The request object is null or not, it is common in both update and create inquiry method so i have created this method instead of calling statement separately for each function .

Reply:
Create a RequestHelper and move this to that class.  If you are using this to validate request data, rename it to isEmptyData(String data). this should be static method there.